scripts/travis_yml_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test to check that travis.yml file & protractor.conf.js have the
+same test suites"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+from core.tests import test_utils
+import python_utils
+import utils
+
+
+def extract_travis_jobs_section():
+    """Extracts the env/jobs section from the .travis.yml file.
+
+    Returns:
+        list. A list of all jobs in travis.yml file.
```

Comment:
Format typeinfo as list(...) and replace "..." with the correct type of an element in the list.

Reply:
Done